sdk/formrecognizer/Azure.AI.FormRecognizer/README.md

```diff
@@ -137,33 +137,78 @@ Recognize data from US sales receipts using a prebuilt model.
 using (FileStream stream = new FileStream(receiptPath, FileMode.Open))
 {
     RecognizedReceiptCollection receipts = await client.StartRecognizeReceipts(stream).WaitForCompletionAsync();
+
     foreach (var receipt in receipts)
     {
-        USReceipt usReceipt = receipt.AsUSReceipt();
+        FormField merchantNameField;
```

Comment:
I like what Python did, where in the Readme they show a generic way to get all the values: https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/formrecognizer/azure-ai-formrecognizer#recognize-receipts
JS decided to go with the approach you show though .... so ....

in the case you decide to go with this approach, consider adding a link to https://westus2.dev.cognitive.microsoft.com/docs/services/form-recognizer-api-v2-preview/operations/GetAnalyzeReceiptResult

Reply:
link comment applies to the other samples

Reply:
I like the approach Python took. Let's go with that.

Reply:
Actually, it's not that simple. Python does not need to do type-checking as we do, and that depends on the field we're extracting.